app/src/comic/test/test_api.py

```
@@ -0,0 +1,131 @@
+
+    def test_write_operation_denied_for_anonymous(self):
+        """測試未認證用戶無法進行寫入操作"""
+        url = reverse("comics-list")
+        payload = {
+            "title_jp": "新作品",
+            "title_tw": "新作品",
+            "author_jp": "作者",
+            "author_tw": "作者",
+            "status_jp": Series.JapanStatus.RUNNING,
+        }
+        # --- 第一部分：測試匿名使用者 (Anonymous) ---
```

Ruff (0.14.7)

Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)